.github/workflows/release.yml

```diff
@@ -85,29 +109,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Setup buildx
-        run: docker buildx create --use
-      - name: build binaries
-        run: docker buildx bake --set *.cache-from=type=gha,scope=buildkit-release-${CRATE} --set *.cache-to=type=gha,scope=buildkit-release-${CRATE} tar-cross
```

Comment:
dropping this means we no longer need https://github.com/containerd/runwasi/blob/main/docker-bake.hcl and the related docker file?

Reply:
Thanks, good catch!